test/webdriver/test-hotkeys.js

```diff
@@ -45,4 +45,46 @@ describe('hotkeys', function() {
     await Driver.waitUntilReduxState(currentState =>
       currentState.pages.pages.length === initialPages.length);
   });
+
+  it('CmdOrCtrl+[1-8]: Select Tab', async function() {
+    const MAX = 8;
+    const state = await Driver.getReduxState();
+    const initialPages = state.pages.pages;
+    expect(initialPages.length).toBeGreaterThan(0, 'Have atleast one tab to start.');
+
+    // Open tabs until we have 8
+    for (let i = initialPages.length; i < MAX; i++) {
```

Comment:
Actually since this test isn't about cmdorctrl+t how about we do the `Driver.ipcSendToMain('synthesize-accelerator', 'CmdOrCtrl+T');` inside the loop and then have a single `await Driver.waitUntilReduxState(currentState => currentState.pages.pages.length === MAX);` after the loop?